vehicles/src/modules/auth/jwt-service-account.strategy.ts

```diff
@@ -0,0 +1,34 @@
+import { ExtractJwt, Strategy } from 'passport-jwt';
+import { PassportStrategy } from '@nestjs/passport';
+import { Injectable } from '@nestjs/common';
+import { passportJwtSecret } from 'jwks-rsa';
+import { AuthService } from './auth.service';
+import { IUserJWT } from 'src/common/interface/user-jwt.interface';
+
+@Injectable()
+export class JwtServiceAccountStrategy extends PassportStrategy(
+  Strategy,
+  'jwt-service-account',
+) {
+  constructor(private authService: AuthService) {
+    super({
+      secretOrKeyProvider: passportJwtSecret({
+        cache: true,
+        rateLimit: true,
+        jwksRequestsPerMinute: 5,
+        jwksUri: `${process.env.KEYCLOAK_ISSUER_URL}/protocol/openid-connect/certs`,
+      }),
+
```

Comment:
`audience` and `issuer` are missing.

Reply:
It was failing when I added audience and issuer, I thought it was something related to how this test is setup.I will retry though.

Reply:
added audience, issuer and algorithm now